test/unit-tests/components/structures/MatrixChat-test.tsx

```diff
@@ -974,6 +1049,22 @@ describe("<MatrixChat />", () => {
                 } as any;
             }
         });
+
+        describe("showScreen", () => {
+            it("should show the 'share' screen", async () => {
+                await getComponent({
+                    initialScreenAfterLogin: { screen: "share", params: { msg: "Hello", format: "B" } },
+                });
+
+                await waitFor(() => {
+                    expect(defaultDispatcher.dispatch).toHaveBeenCalledWith({
+                        action: "share",
+                        msg: "Hello",
+                        format: "B",
```

Comment:
Thank for sorting out tests. what's going on with the `format: "B"` here? shouldn't it be one of the string vals from the SharedFormat enum?

Reply:
True, it is typed so it probably ought to be one of the types.